### PR TITLE
Update Issue templates

### PR DESCRIPTION
**Pull request by gordonfarrell**
Several new, more useful templates

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-template.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+name: Bug Template
+about: Capture bugs with this
+title: ''
+labels: Streamline eCR
```
Comment (BobanL) [inline .github/ISSUE_TEMPLATE/bug-template.md:5]: I think this label should be removed OR this template should be renamed to specify streamline. This also is in task-template

Comment (gordonfarrell) [reply .github/ISSUE_TEMPLATE/bug-template.md:5]: Ahh good catch thanks Boban

Comment (gordonfarrell) [reply .github/ISSUE_TEMPLATE/bug-template.md]: Updated to remove the label

**Review by BobanL — APPROVED** [review]
Sorry for throwing you in a circle with all of these changes 😭

**Pull request merged**